packages/playground/src/samples/ifthenelse.js

```diff
@@ -0,0 +1,75 @@
+module.exports = {
```

Comment:
I think ifthenelse.js is the same example as allofifthenelse.js. Did you mean to change this example?